### DISCUSSION TITLE
Slow View Refresh with 'gg' and 'G' Commands After Updating to LazyVim 14

### DISCUSSION THREAD

**Discussion by serendipitywgy** [LazyVim/LazyVim#5105] [Q&A]
As shown in the video:
https://github.com/user-attachments/assets/034472dc-0116-4c2f-be09-588a49e0053b

**Comment by serendipitywgy** [comment]
I found this question to be a duplicate of "Difference between the newly added "animations" and "scroll", so I closed this question

**Comment by folke** [comment]
Add `vim.g.snacks_animate = false` to your `options.lua`
